lifecycle/lifecycle-runtime-testing/src/commonMain/kotlin/androidx/lifecycle/testing/TestLifecycleOwner.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2024 The Android Open Source Project
```

Comment:
Why is it not moved as-is from Android? Is it a copy from AOSP KMP adoption?

Reply:
yes. I didn't change aosp commit